server.js

```diff
@@ -0,0 +1,88 @@
+const bodyParser = require('body-parser');
+const express = require('express');
+const mongoose = require('mongoose');
+const User = require('./models');
+
+const STATUS_USER_ERROR = 422;
+const STATUS_SERVER_ERROR = 500;
+const server = express();
+
+// allow server to parse JSON bodies from POST/PUT/DELETE requests
+server.use(bodyParser.json());
+
+// TODO: write your server code here
+server.post('/users', (req, res) => {
+  const { christianName, surName } = req.body;
+  if (!christianName || !surName) {
+    res.status(STATUS_USER_ERROR);
+    res.json({ error: 'Need a species or a latinName' });
+    return; // stop here
+  }
+  // create our user save it and send it back as a json response
+  const user = new User({ christianName, surName });
+  user.save((err) => {
+    if (err) {
+      res.status(STATUS_SERVER_ERROR);
+      res.json(err);
+    } else {
+      res.json(user);  //
+    }
+  });
+});
```

Comment:
Make sure to keep consistent spacing.  On line 42 you leave a blank line but here the two routes bump into eachother